src/main/java/com/tabnine/inline/InlineCompletionHandler.java

```diff
@@ -75,14 +68,13 @@ void invoke(@NotNull Project project, @NotNull Editor editor, @NotNull PsiFile f
         final CompletionState completionState = CompletionState.findOrCreateCompletionState(editor);
         int lastDisplayedCompletionIndex = completionState.lastDisplayedCompletionIndex;
 
-        boolean noOldSuggestion = lastDisplayedCompletionIndex == -1 || completionState.prefix == null;
```

Comment:
r u sure?i think this can lead to  a  bug,

Reply:
Afaik the condition was redundant, but just dor the sake of keeping the change as little as possible i can add a check on `oldPrefix`, which is the field that's replacing the `prefix`. Wdyt?

Reply:
i'm not sure, the `completionState.prefix == null` was like "the first time" check(it would be inited in the if below), checking the prefix would not have the same effect